src/Config.js

```diff
@@ -19,13 +19,16 @@ var Config = function (root, configpath) {
     this.initOptions();
     this.initExporters();
     this.initLoaders();
+    this.initRenderers();
     this.initStatics();
     if (!this.configpath) this.ensureDefaultUserConfigPath();
     this.loadUserConfig();
     this.pluginsManager = new PluginsManager(this);  // Do we need back ref?
     this.emit('loaded');
     this.on('server:init', this.attachRoutes.bind(this));
-    this.parsed_opts = {};  // Default. TODO better option management.
+    this.parsed_opts = {
+        renderer: 'carto'
```

Comment:
Yes, I agree. I had to do this mainly for the unit tests, otherwise they were failing. Another library like e.g. commander might be an option, but I think this is a bit out of scope for this PR.